util/keyfile/src/bin/convert_to_pubfile.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+//! Processees pubkeys to file format understood by ingest add_user.
```

Comment:
I'd add parentheses after `pubkeys` and `add_user` to explain what those formats are. Right now I have no idea what this actually does without reading the code and trying to understand what the input format and output format are.